pkg/containerspec/containerspec.go

```diff
@@ -10,9 +10,10 @@ import (
 )
 
 const (
-	defaultCPU    = KubeResourceQuantity("0.1")
-	defaultMemory = KubeResourceQuantity("512")
-	defaultDisk   = KubeResourceQuantity("1024")
+	defaultCPU          = KubeResourceQuantity("0.1")
+	defaultMemory       = KubeResourceQuantity("512")
+	defaultDisk         = KubeResourceQuantity("1024")
+	diskLimitMultiplier = 10
```

Comment:
We use `cpu_burst_add` for a similar goal in paasta, which is an adder, not multiplier.
Maybe it's better to use an added here as well, like 10Gi or so?
Multiplier of 10 to e.g. 20Gi will most likely allow an application to fill up the whole partition and affects the rest of pods running on the same node.

Reply:
I think it is more predictable to use multiplier, because the disk size could be wide range of values; e.g. it will be more obvious to the user that for disk 1GB the default limit is 10GB and for disk 10MB it will be 100MB. If we used an adder then either of these users will be for a surprise.

Reply:
... on the second thought, I agree that 10x overcommit is perhaps too aggressive.

Reply:
... actually, given that we just made disk_limit optional, I think it makes most sense to not use any multiplier at all and do not overcommit by default.